cmd/npmExecuteLint.go

```diff
@@ -62,7 +63,7 @@ func (u *lintUtilsBundle) getGeneralPurposeConfig(configURL string) {
 
 	err = u.FileWrite(filepath.Join(".pipeline", ".eslintrc.json"), content, os.ModePerm)
 	if err != nil {
-		log.Entry().Warnf("failed to write .eslintrc.json file to .pipeline/: %w", err)
+		log.Entry().Warnf("failed to write .eslintrc.json file to .pipeline/: %v", err)
```

Comment:
According to [errorf documentation](https://golang.org/pkg/fmt/#Errorf) %w can be used as a synonym to %v so seems okay to me.